scripts/python_codegen.sh

```diff
@@ -0,0 +1,40 @@
+#! /usr/bin/env bash
+
+# Copyright 2023 SECO Mind Srl
+# SPDX-License-Identifier: Apache-2.0
+
+set -e pipefail
+
+PROTOC_MAJOR='24'
```

Comment:
It might be better to check the minor version as well. There are small changes to the generated files between minors. 
For example, 24.3 and 24.4 generate slightly different files.